### PR TITLE
Update wording in FAQ for AI-codegen guidance

### PR DESCRIPTION
**Pull request by jamieQ**

### CODE REVIEW

**Review by loganblevins — APPROVED** [review]

File: _pages/mobile-interview-project.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ That being said, please do not bulk copy and paste large sections of code from 3
 
  * Q) **Can I use GitHub Copilot, ChatGPT, or other AI code generation tools?**
 
-No. We don't use GitHub Copilot, ChatGPT, or other AI tools for code generation at Block, and we want this project to reflect the same tools and techniques that you would use on the job.
+We'd like the project to be a reflection of your coding skills and approach. You’re welcome to use these tools for minor assistance or inspiration, but please refrain from generating significant portions of your project with them.
```
Comment (kyleve) [inline _pages/mobile-interview-project.md]: How about just

```
...your skills and approach to app development and architecture.
```

Comment (jamieQ) [reply _pages/mobile-interview-project.md]: to clarify, you're proposing just:

> We'd like the project to be a reflection of your skills and approach to app development and architecture.

or like:

> No. We'd like the project to be a reflection of your skills and approach to app development and architecture.

drop all the extra blah blah blah?

Comment (kyleve) [reply _pages/mobile-interview-project.md]: Sorry, 

> We'd like the project to be a reflection of your skills and approach to app development and architecture. You’re welcome to use these tools for minor assistance or inspiration, but please refrain from generating significant portions of your project with them.

Comment (jamieQ) [reply _pages/mobile-interview-project.md]: updated. this is why they pay you the big bucks

**Pull request merged**